src/routes/jaeger.js

```diff
@@ -0,0 +1,57 @@
+import jaeger from "jaeger-client";
```

Comment:
this shouldn't be in routes

Reply:
@bourdakos1 I thought that's where we decided it should go?

Reply:
no I don't think so, it should sit next to app.js
it's not a route

Reply:
Will move this to directory where app.js is

Reply:
Done